### DISCUSSION TITLE
Changing Device A & B

### DISCUSSION THREAD

**Discussion by ghost** [hrvach/deskhop#149] [General]
Hi, so I'm not quite sure if this has been answered yet, so here goes: 

Can I change my Main Device Screen to be on the left? 
So instead of  B | A Screens, I'd have A | B screens? 

I noticed I need my A PC in idle for the keyboard to work (powers off when shutdown).

**Comment by hrvach** [comment]
Yes you can, check the configuration page - settings are here:

![Screenshot 2024-09-06 at 16 43 44](https://github.com/user-attachments/assets/38a601b7-39af-4f31-8e62-a6a2d4989b7c)

The power thing is an unfortunate compromise for simplicity, price and safety. Try finding "Resume by USB" or "Wake on USB" in your BIOS settings, that might keep your USB ports powered while PC is off.

**Reply by ghost** [reply]
Ah, I see my issue now. I wasn't able to load the config off the device using Vivaldi. Only chrome or in my case, ungoogled-chrome did work.

**Comment by hrvach** [comment]
Unfortunately only chrome has webusb/webhid, I use chromium when needed (the open variant) but it would be amazing if Firefox or other alternative browsers started implementing it...
